src/notification.cpp

```diff
@@ -1011,16 +1011,18 @@ void Notification::setSound(const QString &sound)
 
     The name of a sound to be played when the notification is shown.
 
-    This property is transmitted as the hint value "sound-name", names following the freedesktop.org sound
-    naming specification. Sound name can be e.g. "message-new-instant" or "message-new-email".
+    This property is transmitted as the standard \l{https://specifications.freedesktop.org/notification-spec/latest/ar01s08.html}{hint value} "sound-name",
+    names following \l{http://0pointer.de/public/sound-naming-spec.html}{the freedesktop.org sound naming specification}.
+    Sound name can be e.g. "message-new-instant" or "message-new-email".
  */
 /*!
     \property Notification::soundName
 
     The name of a sound to be played when the notification is shown.
 
-    This property is transmitted as the hint value "sound-name", names following the freedesktop.org sound
-    naming specification. Sound name can be e.g. "message-new-instant" or "message-new-email".
+    This property is transmitted as the standard \l{https://specifications.freedesktop.org/notification-spec/latest/ar01s08.html}{hint value} "sound-name",
+    names following \l{http://0pointer.de/public/sound-naming-spec.html}{the freedesktop.org sound naming specification}.
```

Comment:
This could be https://specifications.freedesktop.org/sound-naming-spec/latest/index.html instead, I think.

Reply:
Thanks!

Reply:
It's a little different version though and [freedesktop.org seems to point there too](https://freedesktop.org/wiki/Specifications/sound-theme-spec/), so I guess either is fine. Feel free to resolve without changes and consider this approved. :slightly_smiling_face:

Reply:
Already changed the link. That way you don't have to open a similar bug report after a few years (I hope) :)

Reply:
Looking at the document a bit closer, it seems that the version on fdo is the same 0.3 draft but the version number on overview page has not been updated. Odd.